_posts/en/newsletters/2020-04-08-newsletter.md

```diff
@@ -0,0 +1,188 @@
+---
+title: 'Bitcoin Optech Newsletter #92'
+permalink: /en/newsletters/2020/04/08/
+name: 2020-04-08-newsletter
+slug: 2020-04-08-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter contains a list of notable changes to popular
+Bitcoin infrastructure projects.  FIXME: mention_PR_club_section_if_added
+
+## Action items
+
+*None this week.*
+
+## News
+
+<!-- Sources like BOLT0 and BIP199 expand the "H" in HTLC to "Hashed",
+but I think it makes more sense to drop the suffix here for congruency
+with "Point" in PTLC. -->
+
+- **Work on PTLCs for LN using simplified ECDSA adaptor signatures:**
+  Point Time Locked Contracts (PTLCs) are an alternative to the Hash
+  Time Locked Contracts (HTLCs) currently used to enable routable
+  payments in LN.  A problem with the existing HTLC mechanism is that
+  every hop along a payments path secures its conditional payment with
+  the same hash digest.  This means that a user who controls two nodes
+  along the same path knows that any hops between those nodes were not
+  the ultimate spender or receiver of the payment.  This not only
+  reduces the amount of privacy provided by LN's onion routing but it
+  also allows a malicious user to steal the routing fees paid to the
+  in-between hops (this is known as the [wormhole attack][]).  For
+  example, in the following route, Mallory can steal Bob and Carol's
+  routing fees as well as conclude that neither of them is the spender
+  or receiver of the ultimate payment.
+
+      Alice → Mallory → Bob → Carol → Mallory' → Dan
+
+    PTLCs make it possible for each hop to use a different identifier
+    for the payment by using adaptor signatures (which represent
+    *points* on an elliptic curve) rather than hashes.  Adaptor
+    signatures were originally described for use with the [schnorr
+    signature scheme][topic schnorr signatures].  It's known to be
+    possible to use them with Bitcoin's current ECDSA signature scheme
+    (see [Newsletter #16][news16 2pecdsa scriptless]) but the process
+    relies on two-party ECDSA signing (2pECDSA) which is complex and
+    requires security assumptions beyond those normally required for
+    Bitcoin-style ECDSA signatures.  However, more recently, Lloyd
+    Fournier published a [paper][fournier otves] describing how to
+    securely use adaptor signatures using just regular 2-of-2 Bitcoin
+    multisig (e.g.  `OP_CHECKMULTISIG`) and simple discrete log
+    equivalence (DLEQ); this was summarized in a [post][uSEkaCIO email]
+    to the Lightning-Dev mailing list last November.
+
+    Last week during the [Lightning HackSprint][], several developers
+    [worked][ptlc challenge] on implementing these 2-of-2 multisig
+    adaptor signatures.  The result was rough code for it being written
+    for the C-language [libsecp256k1][jonasnick otves] and Scala [bitcoin-s][nkohen otves] libraries.
+    This is currently nowhere near ready for production, but it can help
```

Comment:
nit: "currently nowhere near" is a bit informal, and implies to me that the developers are actively planning to launch this in production in the future. Perhaps "this is not suitable for production, but ..."